README.md

```diff
@@ -3,6 +3,7 @@
 [![NPM Version](https://img.shields.io/npm/v/get-js.svg?style=flat-square)](https://www.npmjs.com/package/get-js)
 [![NPM Downloads](https://img.shields.io/npm/dt/get-js.svg?style=flat-square)](https://www.npmjs.com/package/get-js)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg?style=flat-square)](LICENSE)
+[![Build Status](https://travis-ci.org/beinganonymous/get-js.svg?branch=master)](https://travis-ci.org/beinganonymous/get-js)
```

Comment:
This uses the badge for your forked repository. It will need this badge instead.
```
![Build Status](https://img.shields.io/travis/kabirbaidhya/get-js?style=flat-square)
```